dataset/data_test.go

```diff
@@ -139,7 +146,8 @@ func expectedData(isEmpty bool) string {
 		"Related Links: [{related dataset url related dataset title}]\n" +
 		"Canonical Topic: canonicalTopicID\n" +
 		"Subtopics: [secondaryTopic1ID secondaryTopic2ID]\n" +
-		"Survey: census\n"
+		"Survey: census\n" +
+		"Related Content: [{related content description related content url related content title}]\n"
```

Comment:
I know that this is not in these changes but the writeToFile function, below this line, should probably be deleted as it is not being used.

Reply:
true although from reading the comment above it it looks like it might be useful in debugging?

Reply:
Yes, I did notice the comment, but I'm of the view that code should not be left in for debugging purposes alone. Debugging comments should be removed when done with. And a function like that should be stored somewhere else, such as in documentation, if it's useful to keep.